terraform/environment/region/modules/uploads_s3_bucket/s3_bucket_lifecycles.tf

```diff
@@ -18,5 +18,22 @@ resource "aws_s3_bucket_lifecycle_configuration" "main" {
     status = "Enabled"
   }
 
+  rule {
+    id = "delete-objects-replicated-to-sirius"
+
+    filter {
```

Comment:
should we use a different filter, as the replicate tag is what we put on so doesn't mean it has been replicated?

Reply:
We can only filter based on object prefix, tags or a combination.

We will be notified for each object that fails replication on the first attempt (currently within 15 minutes of the replicate tag being set).

Expiring objects after 30 days gives the chance to respond (manually at first automatically later if necessary).